src/documents/APIChangesFrom2_0To3_0.md

```diff
@@ -19,10 +19,56 @@ This section describes major APIs that were in common use. For information about
 
 ### Preferences Framework ###
 
+#### Model Changes ####
+
+##### Reset ######
+By default, reset will only reset the `preferences` model path. Other model values will remain unchanged. If you'd like to also have these paths changed, add a listener to the `beforeReset` event to execute a fireChangeRequest for the model paths you need to reset. (See: ArrowScrolling.js)
+
+```javascript
+...
+listeners: {
+    "beforeReset.resetPanelIndex": {
+        listener: "{that}.applier.fireChangeRequest",
+        args: {path: "panelIndex", value: 0, type: "ADD", source: "reset"}
+    }
+}
+...
+```
+
+##### Model Paths ######
+
+Any prefsEditor using the `fluid.prefs.arrowScrolling` grade, such as the one contained in `fluid.prefs.separatedPanel`, will contain the following new model paths.
+
+<table>
+    <thead>
+        <tr>
+            <th>Model Path</th>
+            <th>Default</th>
+            <th>Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><code>panelIndex</code></td>
+            <td>0</td>
+            <td>The index of the panel to scroll to in the small screen view.</td>
+        </tr>
+        <tr>
+            <td><code>panelMaxIndex</code></td>
+            <td>A number</td>
```

Comment:
Is it possible to be more specific with what this number is, such as "the total number of panels"?